src/components/basic/containers/Scrollable.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import classnames from "classnames";
+
+const Scrollable = props => {
+  const { children, className, ...rest } = props;
+  const cssClasses = classnames("dg_scrollable", className);
```

Comment:
This is subjective, but wondering if something like `dg_scroll-container` or something along those lines would work. Scrollable seems like it could be vague to someone just coming in.

Reply:
@theilman7 haha, thats waht i orginally had, will update